pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -137,10 +138,20 @@ func validateClusterServiceBrokerSpec(spec *sc.ClusterServiceBrokerSpec, fldPath
 	if spec.RelistRequests < 0 {
 		allErrs = append(
 			allErrs,
-			field.Required(fldPath.Child("relistRequests"), "relistDuration must be greater than zero"),
+			field.Required(fldPath.Child("relistRequests"), "relistRequests must be greater than zero"),
 		)
 	}
 
+	if spec.RelistDuration != nil {
+		zeroDuration := metav1.Duration{0}
+		if spec.RelistDuration.Duration <= zeroDuration.Duration {
```

Comment:
since you don't use zeroDuration anywhere else, can you just do this:
if spec.RelistDuration.Duraiton <= metav1.Duration{0} {

Reply:
I actually had that initially and was surprised to find syntax errors; not immediately clear why that's the case but I'm sure there's some kind of parsing rule I'm not aware of: 

```
if spec.RelistDuration.Duration <= metav1.Duration{0}.Duration {
...
pkg/apis/servicecatalog/validation/broker.go:146: syntax error: unexpected { at end of statement
pkg/apis/servicecatalog/validation/broker.go:154: syntax error: non-declaration statement outside function body
```

Looks like the compiler is happy with this:

`if (spec.RelistDuration.Duration <= metav1.Duration{0}.Duration) {`

Any preference?

Reply:
How about convert it to an int and compare with 0?

Reply:
I'm fine with leaving as is, I was just curious. I think leaving it as is clear. Thanks!